david_vicklund/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var mocha = require('gulp-mocha');
+
+var tasks = {
+  default: function() {
+    return gulp.src('./test/test.js')
+      .pipe(mocha({reporter: 'nyan'}));
+  },
+
+  jshint: function() {
+    return gulp.src([
+      'gulpfile.js',
+      'server.js',
+      './test/test.js'
+    ]).pipe(jshint())
+      .pipe(jshint.reporter('default'))
+      .on('error', function(error) {
+        console.log(error);
+      });
+  }
+};
+
+gulp.task('default', tasks.default);
```

Comment:
default should go last and should probably containt jshint and mocha tasks. this looks like default only runs mocha tests.